lib/components/Images/image.css

```diff
@@ -0,0 +1,33 @@
+.image-component {
+    display: flex;
+    flex-flow: column;
+    gap: 1rem;
+}
+
+.image-component iframe{
+    height: 8rem;
+}
+
+.container-img {
+    padding: 1rem;
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    border-radius: 1rem;
```

Comment:
give a specific width and height here, not in images itself.

Reply:
I understood your point and implemented it.